.changeset/sweet-trainers-clean.md

```diff
@@ -0,0 +1,13 @@
+---
+"@builder.io/react": patch
+"@builder.io/sdk-angular": patch
+"@builder.io/sdk-react-nextjs": patch
+"@builder.io/sdk-qwik": patch
+"@builder.io/sdk-react": patch
+"@builder.io/sdk-react-native": patch
+"@builder.io/sdk-solid": patch
+"@builder.io/sdk-svelte": patch
+"@builder.io/sdk-vue": patch
```

Comment:
oh sorry i forgot this, we don't yet have Forms support in next.js sdk and react-native sdk so we can safely remove them from the changesets to avoid confusions
https://github.com/BuilderIO/builder/blob/main/packages/sdks/README.md#builder-blocks

```suggestion
"@builder.io/react": patch
"@builder.io/sdk-angular": patch
"@builder.io/sdk-qwik": patch
"@builder.io/sdk-react": patch
"@builder.io/sdk-solid": patch
"@builder.io/sdk-svelte": patch
"@builder.io/sdk-vue": patch
```